inc/customizer/extend-custom-controls/src/color-group/color-group-component.js

```diff
@@ -1,4 +1,9 @@
 import PropTypes from 'prop-types';
+const {__} = wp.i18n;
+import AstraColorPickerControl from '../common/astra-color-picker-control';
+const {Tooltip} = wp.components;
+import {Dashicon} from '@wordpress/components';
```

Comment:
```suggestion
import {Dashicon, Tooltip} from '@wordpress/components';
```

This both packages should be correct, right? Can you confirm this?

Reply:
Right ✔️ Corrected as per suggestion.